crates/bevy_gizmos/src/lib.rs

```diff
@@ -89,7 +92,8 @@ impl Plugin for GizmoPlugin {
                 (
                     draw_aabbs,
                     draw_all_aabbs.run_if(|config: Res<GizmoConfig>| config.aabb.draw_all),
-                ),
+                )
+                    .after(TransformSystem::TransformPropagate),
```

Comment:
So, instead of adding the `draw_aabbs` systems in `Update`, you should add them to `PostUpdate`. Otherwise this fixes nothing.

Reply:
updated